lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1676,6 +1676,42 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
       return b.create<arith::SubIOp>(loc, lhs, scaled);
     }
   }
+  if (auto subScalar = dyn_cast<AtenSubScalarOp>(op)) {
+    Type dtype = converter->convertType(subScalar.getType())
+                     .cast<RankedTensorType>()
+                     .getElementType();
+    Value self = convertScalarToDtype(b, loc, payloadArgs[0], dtype);
+    Value other = convertScalarToDtype(b, loc, operands[1], dtype);
+    Value alpha = convertScalarToDtype(b, loc, operands[2], dtype);
+    if (dtype.isa<mlir::FloatType>()) {
+      Value mult = b.create<arith::MulFOp>(loc, self, alpha);
```

Comment:
`alpha` should be multiplied by `other`, not `self`. 

https://pytorch.org/docs/stable/generated/torch.sub.html

Also, make sure to add a unit test to check that the `alpha` logic is correct